plugins/dummy_sha256_sdk/Cargo.toml

```diff
@@ -10,3 +10,6 @@ clap = { version = "4.5.18", features = ["derive"] }
 hipcheck-sdk = { path = "../../sdk/rust" }
 sha2 = "0.10.8"
 tokio = { version = "1.40.0", features = ["rt"] }
+hipcheck-sdk-macros = { path = "../../hipcheck-sdk-macros", version = "0.1.0" }
+serde_json = "1.0.128"
+schemars = "0.8.21"
```

Comment:
one of the things Andrew mentioned wanting was for `hipcheck_sdk` to re-export the `hipcheck_sdk_macros` proc macro. Please try to make this work, and then see if we can make it so `serde_json` and `schemars` are not also explicit dependencies (add anything we need for the macro to the `hipcheck_sdk::prelude` module maybe?)